buidl/test/test_ecc.py

```diff
@@ -1,248 +1,42 @@
-        # (47,71) + (117,141)
-        # (143,98) + (76,66)
-        prime = 223
-        a = FieldElement(0, prime)
-        b = FieldElement(7, prime)
-
-        additions = (
-            # (x1, y1, x2, y2, x3, y3)
-            (192, 105, 17, 56, 170, 142),
-            (47, 71, 117, 141, 60, 139),
-            (143, 98, 76, 66, 47, 71),
-        )
-        # iterate over the additions
-        for x1_raw, y1_raw, x2_raw, y2_raw, x3_raw, y3_raw in additions:
-            # Initialize points this way:
-            # x1 = FieldElement(x1_raw, prime)
-            # y1 = FieldElement(y1_raw, prime)
-            # p1 = Point(x1, y1, a, b)
-            # x2 = FieldElement(x2_raw, prime)
-            # y2 = FieldElement(y2_raw, prime)
-            # p2 = Point(x2, y2, a, b)
-            # x3 = FieldElement(x3_raw, prime)
-            # y3 = FieldElement(y3_raw, prime)
-            # p3 = Point(x3, y3, a, b)
-            x1 = FieldElement(x1_raw, prime)
-            y1 = FieldElement(y1_raw, prime)
-            p1 = Point(x1, y1, a, b)
-            x2 = FieldElement(x2_raw, prime)
-            y2 = FieldElement(y2_raw, prime)
-            p2 = Point(x2, y2, a, b)
-            x3 = FieldElement(x3_raw, prime)
-            y3 = FieldElement(y3_raw, prime)
-            p3 = Point(x3, y3, a, b)
-            # check that p1 + p2 == p3
-            self.assertEqual(p1 + p2, p3)
-
-    def test_rmul(self):
-        # tests the following scalar multiplications
-        # 2*(192,105)
-        # 2*(143,98)
-        # 2*(47,71)
-        # 4*(47,71)
-        # 8*(47,71)
-        # 21*(47,71)
-        prime = 223
-        a = FieldElement(0, prime)
-        b = FieldElement(7, prime)
-
-        multiplications = (
-            # (coefficient, x1, y1, x2, y2)
-            (2, 192, 105, 49, 71),
-            (2, 143, 98, 64, 168),
-            (2, 47, 71, 36, 111),
-            (4, 47, 71, 194, 51),
-            (8, 47, 71, 116, 55),
-            (21, 47, 71, None, None),
-        )
-
-        # iterate over the multiplications
-        for s, x1_raw, y1_raw, x2_raw, y2_raw in multiplications:
-            # Initialize points this way:
-            # x1 = FieldElement(x1_raw, prime)
-            # y1 = FieldElement(y1_raw, prime)
-            # p1 = Point(x1, y1, a, b)
-            x1 = FieldElement(x1_raw, prime)
-            y1 = FieldElement(y1_raw, prime)
-            p1 = Point(x1, y1, a, b)
-            # initialize the second point based on whether it's the point at infinity
-            # x2 = FieldElement(x2_raw, prime)
-            # y2 = FieldElement(y2_raw, prime)
-            # p2 = Point(x2, y2, a, b)
-            if x2_raw is None:
-                p2 = Point(None, None, a, b)
-            else:
-                x2 = FieldElement(x2_raw, prime)
-                y2 = FieldElement(y2_raw, prime)
-                p2 = Point(x2, y2, a, b)
-            # check that the product is equal to the expected point
-            self.assertEqual(s * p1, p2)
-
-
```

Comment:
Do you really want to get rid of all these tests? I suggest moving to a `test_pecc.py` file. I could see potentially adding a flag in the future to disable running it if you're using `cecc.py`.

Reply:
Deleting these test cases seemed like an oversight so I added them back in:
https://github.com/buidl-bitcoin/buidl-python/pull/2